queries/python/injections.scm

```diff
@@ -4,3 +4,17 @@
   arguments: (argument_list (string) @regex))
  (#eq? @_re "re")
  (#match? @regex "^r.*"))
+
+; Class docstring
+((class_definition
+  body: (block . (expression_statement (string) @rst)))
+ (#offset! @rst 0 3 0 -3))
```

Comment:
Shouldn't you check here for triple strings? This would balso match strings with single quotes.

Reply:
I added `(#match? @rst "^[\"']{3}")`, but looks like it does't play nice with offset, it started to recognize just the first line.

Reply:
Created https://github.com/nvim-treesitter/nvim-treesitter/issues/923 to keep track of this